### PR TITLE
Fix null in concat

### PR DESCRIPTION
**Pull request by KevinCorcoran**

### CODE REVIEW

**Comment by cprice404** [comment]
+1

**Comment by cprice404** [comment]
@fpringvaldsen I'm going to go ahead and merge this because it looks fairly innocuous to me and I think it will fix a bug I'm hitting.  Can you review it posthumously tomorrow and let us know if you see anything awry?

File: lib/hocon/impl/config_null.rb [hunk 1]
```diff
@@ -16,12 +16,12 @@ def transform_to_string
     "null"
   end
 
-  def render(sb, indent, atRoot, options)
-    sb.append("null")
+  def render_value_to_sb(sb, indent, at_root, options)
+    sb << "null"
   end
 
   def newCopy(origin)
```
Comment (cprice404) [inline lib/hocon/impl/config_null.rb]: this should be `new_copy` (was already wrong, not part of your PR).  fixed in my local branch.

Comment (KevinCorcoran) [inline lib/hocon/impl/config_null.rb]: okay, thanks

**Pull request merged**